Dockerfile

```
@@ -0,0 +1,20 @@
+# pull official base image
+FROM node:22.3.0 as dev
+
+# set working directory
+WORKDIR /syncmaster_ui
+
+# install app dependencies
+COPY package*.json .
+RUN npm cache clean --force \
+    npm install --global yarn
+RUN npm install
```

```suggestion
RUN npm cache clean --force \
    npm install --global yarn

COPY package*.json .
RUN npm ci
```

BTW, почему бы не добавить yarn в зависимости?

---

я бы этим занимался в отдельной задаче после этих трех мров, потому что сейчас не проверишь сборку образа ломается всё, они все связаны

---

Ок